opus/application/apps/ui/templates/ui/base.html

```diff
@@ -336,6 +340,29 @@ <h3 class='op-header-text'></h3>
             target="op-update-url"
             yes_text="Keep Message"
             no_text="Discard Message"
+            allow_close=True
+        %}
+        {% include "ui/confirm_modal.html" with
+            id_name="op-browser-version-msg"
+            confirm_title="Message About Your Browser Version"
+            confirm_msg="Please update your browser. OPUS supports Firefox (66+), Chrome (74+), Safari (12.1+), Opera (58+), and Microsoft Edge (18+)."
+            target="op-browser-version-msg"
+            allow_close=False
+        %}
+        {% include "ui/confirm_modal.html" with
+            id_name="op-browser-size-msg"
+            confirm_title="Message About Your Browser Size"
+            confirm_msg="Please resize your browser. OPUS works best with a browser width of at least 1280px."
```

Comment:
Also let's spell out the word "pixels" here. Nobody knows what a "px" is. Although pixels is not really technically accurate, at least people will have heard of it.